### PR TITLE
[FEAT] Adding ember-angle-brackets-codemod to manifest

### LINKED ISSUE #15: Add ember-angle-brackets-codemod
Add the ember-angle-brackets-codemod to the manifest for ember-cli version 3.10 onwards.

### PR DESCRIPTION
**Pull request by rajasegar**
Fixes #15

### CODE REVIEW

**Comment by rwjblue** [comment]
I’m not quite sure the codemod is ready to push onto most users just yet. I’m worried about the various issues reported over in the codemod repo, but I’d definitely like to get a general agreement on what needs to happen to land this PR (and therefore “officially” recommend the codemod).

File: manifest.json [hunk 1]
```diff
@@ -41,5 +41,13 @@
...
     "nodeVersion": "6.0.0",
     "commands": ["jscodeshift -t https://raw.githubusercontent.com/simplabs/qunit-dom-codemod/master/qunit-dom-codemod.js ./tests"]
+  },
+  "ember-angle-brackets-codemod": {
+    "versions": {
+      "ember-cli": "3.10.0-beta.1"
+    },
+    "projectOptions": ["app", "addon"],
+    "nodeVersion": "6.0.0",
+    "commands" : ["npx ember-angle-brackets-codemod angle-brackets ./app/templates"]
```
Comment (kellyselden) [inline manifest.json]: npx is implied, so you can remove that part.

**Comment by kellyselden** [comment]
Is there anyone we can invite to this issue to come to a "recommended" conclusion?

**Comment by rwjblue** [comment]
Not really sure, I think we just need to keep working through the issues over in the repo. The team has made great strides in the last couple months, and I think it should be possible to move forward soonish...

File: manifest.json (hunk 1 above)
Comment (rajasegar) [reply manifest.json]: Done

File: manifest.json [hunk 2]
```diff
@@ -41,5 +41,13 @@
     "projectOptions": ["app", "addon"],
     "nodeVersion": "6.0.0",
     "commands": ["jscodeshift -t https://raw.githubusercontent.com/simplabs/qunit-dom-codemod/master/qunit-dom-codemod.js ./tests"]
+  },
+  "ember-angle-brackets-codemod": {
+    "versions": {
+      "ember-cli": "3.10.0-beta.1"
+    },
+    "projectOptions": ["app", "addon"],
+    "nodeVersion": "6.0.0",
```
Comment (kellyselden) [inline manifest.json]: Judging by https://github.com/ember-codemods/ember-angle-brackets-codemod/commit/b45c923e7167ee6b6c83fa09ac7fa3211988fc21#diff-354f30a63fb0907d4ad57269548329e3R3, I think this should be 8.0.0

File: manifest.json [hunk 3]
```diff
@@ -41,5 +41,13 @@
     "projectOptions": ["app", "addon"],
     "nodeVersion": "6.0.0",
     "commands": ["jscodeshift -t https://raw.githubusercontent.com/simplabs/qunit-dom-codemod/master/qunit-dom-codemod.js ./tests"]
+  },
+  "ember-angle-brackets-codemod": {
+    "versions": {
+      "ember-cli": "3.10.0-beta.1"
```
Comment (kellyselden) [inline manifest.json]: According to https://github.com/rwjblue/ember-angle-bracket-invocation-polyfill/blob/315403c38d3520f96b4f2f067eda285725cb8826/index.js#L16, I think this should be `"ember-source": "3.10.0-alpha.1"`

**Comment by rajasegar** [comment]
@kellyselden Oops, My bad, I overlooked the comments, changed now

**Comment by kellyselden** [comment]
This looks good on our end. Tested on https://github.com/ember-cli/ember-cli-update/pull/606, but there appear to be a couple remaining issues with the codemod that @tylerturdenpants is working on in https://github.com/ember-codemods/ember-angle-brackets-codemod/pull/97. Perhaps he can keep us in the loop here?

**Comment by tylerturdenpants** [comment]
@kellyselden @rwjblue I think I might be able to get the PR to be green for a few remaining issues that are not `ember-template-recast` related. There are some test that are failing strictly due to formatting, so maybe all we need to do is reach consensus on those. I’ll work on it again today and hopefully be able to check back in with some progress.

File: manifest.json [hunk 4]
```diff
@@ -90,6 +90,14 @@
     "nodeVersion": "6.0.0",
     "commands": ["ember-3x-codemods cp-property-map app/**/*.js"]
   },
+  "ember-angle-brackets-codemod": {
+    "versions": {
+      "ember-source": "3.10.0-alpha.1"
```
Comment (kellyselden) [inline manifest.json]: This version doesn't exist. We should probably use 3.10.0-beta.1.

Comment (rajasegar) [reply manifest.json]: Updated

**Pull request merged**